graphql-dgs-codegen-gradle/src/main/kotlin/com/netflix/graphql/dgs/codegen/gradle/ClientUtilsConventions.kt

```diff
@@ -47,14 +47,26 @@ object ClientUtilsConventions {
             configurationDependencies.add(project.dependencies.create(dependencyString))
             logger.info("DGS CodeGen added [{}] to the {} dependencies.", dependencyString, dependencyConfiguration)
 
-            project.dependencyLocking.ignoredDependencies.add(dependencyLockString)
-            logger.info("DGS CodeGen added [{}] to the ignoredDependencies.", dependencyLockString, dependencyConfiguration)
+            try {
+                project.dependencyLocking.ignoredDependencies.add(dependencyLockString)
+                logger.info(
+                    "DGS CodeGen added [{}] to ignoredDependencies.",
+                    dependencyLockString
+                )
+            } catch (e: Exception) {
+                // do nothing, this is supplemental and seems to work in certain contexts but not in others
+                logger.info(
+                    "Failed to add DGS CodeGen to ignoredDependencies because: {}",
+                    dependencyLockString,
+                    e
+                )
+            }
```

Comment:
I kept this here in case the plugin's initialization changes at any time in the lifecycle, as an extra safety net.